packages/@okta/vuepress-site/docs/guides/add-an-external-idp/index.md

```diff
@@ -1,5 +1,8 @@
 ---
 title: Add an external Identity Provider
+meta:
+  - name: description
+    content: Okta supports authentication with an external OpenID Connect and SAML Identity Providers. Get an overview of the process and prerequisites, and the instructions required to set one up.
```

Comment:
Okta supports authentication with external OpenID Connect Identity Providers as well as SAML. Get an overview of the process and prerequisites, and the instructions required to set one up.

Reply:
Good catch - thanks!